### DISCUSSION TITLE
Headstart to fix hot start in editor tester

### DISCUSSION THREAD

**Discussion by ChromaCat248** [ppy/osu#29271] [Ideas]
Currently, testing a section of the map sends you headfirst wherever you put the play head without any time to react, which makes testing a part of the map inconvenient when it has things already mapped just before it.

I'd suggest either a countdown like when you pause in taiko/catch/mania, or play the previous second or so without any of the hit objects like Beat Saber's practice mode does.

idk whether to consider this a bug or not so i'm putting it here

**Comment by ziv-vy** [comment]
https://github.com/ppy/osu/issues/24607
